modules/ROOT/pages/provisioning-hyperv.adoc

```diff
@@ -0,0 +1,104 @@
+= Provisioning Fedora CoreOS on Microsoft HyperV
+
+This guide shows how to provision new Fedora CoreOS (FCOS) nodes on Microsoft's HyperV.
+
+== Prerequisites
+
+You must have an Ignition configuration file containing your customizations. If you do not have one, see xref:producing-ign.adoc[Producing an Ignition File]
+You will also need a small utility from the libhvee https://github.com/containers/libhvee called `kvpctl`. It "injects" your Ignition file into your virtual machine. Precompiled binaries can be found on the project's https://github.com/containers/libhvee/releases[release page].
+
+=== Downloading the HDX image
+
+HDX images can be manually downloaded from the download page.  Be sure to decompress the image after downloading.
+
+== Booting a new VM on Microsoft HyperV
+
+=== Creating a virtual machine
+
+There are numerous ways to create a virtual machine in HyperV.  This approach uses the HyperV Manager and its new virtual machine wizard. Once you have the HyperV Manager graphic interface running, right-click on _New_ as seen in the following image.
+
+image::new_hyperv.png[HyperV New Virtual Machine Wizard]
+
+This will launch the _New Virtual Machine Wizard_.  You can follow the guided wizard but take note of the following two exceptions.
+
+==== Configuring networking
+
+When asked about the networking in the _New Virtual Machine Wizard_, be sure set up networking or your virtual machine will not be able to communicate with anything.  It is sufficient to select the _Default_ Networking option.
+
+==== Configuring the Fedora CoreOS image as the disk
+You need to assign the HDX disk image you downloaded earlier as the virtual machine's primary disk. This is not the default setting.  You must click _Use an existing virtual disk_ and then specify the location of the image you downloaded earlier.  See the image below for an example.
+
+image::hyperv-disk.png[]
```

Comment:
```suggestion
image::hyperv-disk.png[Hyper-V Virtual Machine Disk Wizard]
```

I think it's good form to provide an alt-text with the images to improve accessibility.